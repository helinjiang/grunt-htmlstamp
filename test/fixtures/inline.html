<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title></title>
    <link rel="stylesheet" href="./test1.css">
    <script type="text/javascript" src="./test1.js"></script>
    <script type="text/javascript" src="./test2.js"></script>
    <script type="text/javascript"></script>
</head>
<body>

</body>
</html>
