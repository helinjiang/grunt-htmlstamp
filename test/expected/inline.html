<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title></title>
    <style>body {
        color: red;
    }</style>
    <script type="text/javascript">/**
     * [文件描述]
     *
     * @author linjianghe
     * @date 2015/11/5
     */
    console.log("hello,test1.js!");</script>
    <script type="text/javascript" src="./test2.js"></script>
    <script type="text/javascript"></script>
</head>
<body>

</body>
</html>